File: accommodationService/src/main/java/com/example/accommodationService/model/Address.java
package com.example.accommodationService.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Generated;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Address {
    @Generated
    @Id
    private String id;
    private String streetName;
    private String streetNumber;
    private String postalCode;
    private String city;
    private String country;
}

File: BookingAccomodation/Accomodations/Model/Address.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace Accomodations.Model
{
    public class Address
    {
        [BsonRepresentation(BsonType.String)]
        public Guid Id { get; set; }
        public String StreetName { get; set; }
        public String StreetNumber { get; set; }
        public String PostalCode { get; set; }
        public String City { get; set; }
        public String Country { get; set; }
    }
}
